coops-ui/src/components/Dashboard/DirectorListSm.vue

```diff
@@ -1,71 +1,114 @@
 <template>
-  <ul class="list list-container">
-    <li class="list-item" v-for="director in directors" v-bind:key="director.id">
-      <v-avatar color="primary" size="25">
-        <span class="white--text small">{{ director.officer.firstName.substring(0,1)}}</span>
-      </v-avatar>
-      <div class="director-info">
+  <v-expansion-panels class="list list-container" :accordion=true :multiple=true>
```

Comment:
Normally, a prop's value should be in quotes, eg, "true".
Here, since these props aren't reactive, you don't need the colon.
But really, in this case, the props just needs to exist, so you can just write "accordion multiple".

Reply:
Interesting, i will give it a try and change it!

Reply:
That's how the other props work (eg, filled, flat, etc).

Reply:
cool, i like it!